File: php/classes/Base64.php
<?php

class Base64 
{
	/**
	* Encodes text to base64 string.
	*/	
	public static function encode($text)
	{
		return base64_encode($text);
	}
	
	/**
	* Decodes base64 string to text.
	*/
	public static function decode($base64)
	{
		return base64_decode($base64);
	}
	
}

?>

File: csharp/ASCrypt/Base64.cs
using System;
using System.Security.Cryptography;

namespace ASCrypt
{
    public class Base64
    {
        /// <summary>
        /// Encodes bytes to a base64 string.
        /// </summary>
        public static String Encode(Byte[] bytes)
        {
            return Convert.ToBase64String(bytes);
        }

        /// <summary>
        /// Decodes base64 string to bytes.
        /// </summary>
        public static Byte[] Decode(String base64)
        {
            return Convert.FromBase64String(base64);
        }

    }

}
